local-cli/generate-windows.js

```diff
@@ -10,18 +11,23 @@ var yeoman = require('yeoman-environment');
  * @param  {String} projectDir root project directory (i.e. contains index.js)
  * @param  {String} name       name of the root JS module for this app
  */
-function generateWindows (projectDir, name) {
+function generateWindows (projectDir, name, ns) {
   var oldCwd = process.cwd();
+
+  var exists = fs.existsSync(projectDir);
+  exists && childProcess.execSync('rmdir '+ projectDir + ' /s /q');
+  !exists && fs.mkdirSync(projectDir);
```

Comment:
I think it might be better to just bail if the fs.rmdir operation fails.  This assumes that the user must create the windows project from a Windows command environment, which is not necessarily going to be the case.  I can foresee us wanting to support Linux users to initialize Windows projects and potentially run cloud builds.